.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: php
+dist: xenial
+os: linux
```

Comment:
These root keys were defaulting to the configured values so I have just added them here to address all configuration validation issues reported by Travis CI